data_diff/database.py

```diff
@@ -9,6 +9,8 @@
 from concurrent.futures import ThreadPoolExecutor
 import threading
 from typing import Dict
+from cryptography.hazmat.backends import default_backend
```

Comment:
There is no guarantee that this package exists.

1) Included in pyproject for when running `pip install data-diff[snowflake]`

2) Imported only when snowflake is loaded. (See `import_snowflake()`)